pkg/cmd/root/help_topic.go

```diff
@@ -86,8 +86,12 @@ var HelpTopics = []helpTopic{
 			available in the viewport. When the value is a percentage, it will be applied against
 			the number of columns available in the current viewport.
 
-			%[1]sGH_NO_UPDATE_NOTIFIER%[1]s: set to any value to disable update notifications. By default, gh
-			checks for new releases once every 24 hours and displays an upgrade notice on standard
+			%[1]sGH_NO_UPDATE_NOTIFIER%[1]s: set to any value to disable GitHub CLI update notifications.
+			By default, gh checks for new releases once every 24 hours and displays an upgrade notice on standard
+			error if a newer version was found.
+
+			%[1]sGH_NO_EXTENSION_UPDATE_NOTIFIER%[1]s: set to any value to disable GitHub CLI extension update notifications.
```

Comment:
I think I'd expect to see something in the `gh ext --help` indicating extension upgrade behaviour, and maybe pointing to this env var.